.github/workflows/ci.yml

```
@@ -0,0 +1,57 @@
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  windows-latest:
+    name: Windows
+    runs-on: windows-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)